cluster-sync/sync.sh

```diff
@@ -27,7 +27,7 @@ for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
 done
 
 registry=${IMAGE_REGISTRY:-localhost:$(_port registry)}
-DOCKER_REPO=${registry} make push
+DOCKER_REPO=${registry} BUILDAH_PUSH_FLAGS="--tls-verify=false" make manifest manifest-push
```

Comment:
I understand why you added the tls-verify=false, because we can't really verify the identity of the registry:5000 in the kubevirtci cluster. Can we only add this flag if we are in that particular situation. I would really like to verify the identity when I am pushing an actual release to quay.io for instance.

Reply:
Done